bundles/binding/org.openhab.binding.tcp/src/main/java/org/openhab/binding/tcp/protocol/ProtocolBindingProvider.java

```diff
@@ -40,7 +40,7 @@
 	/* Get a list of the openHAB Commands associated with the combination of the given Item and protocol command */
 	public List<String> getItemNames(String protocolCommand);
 
-	/* Get a list of the openHAB Commands associated with the combingation of the given Item and protocol command */
+	/* Get a list of the openHAB Commands associated with the combination of the given Item and protocol command */

```

Comment:
this changes doesn't seem to be related to NIBE so please create another PR for them …